paper/paper.md

```diff
@@ -1,63 +1,54 @@
 ---
-title: 'JointSurvivalModels.jl: Numeric approach to joint models'
+title: 'JointSurvivalModels.jl: A Julia package for general Bayesian joint models'
```

Comment:
now we have the `Survival` in the package name, I wonder if we should adjust the title.
E.g. "A Julia package for joint modeling of survival and longitudinal data"
so that we get "survival" in the title. I think "Bayesian" could be optional before "joint". Could we also use the package to use maximum likelihood estimation? or not? (that would also be good to mention in the paper)

Reply:
I changed it to "A Julia package for Bayesian joint modeling of survival and longitudinal data" which just fits on two lines in the pdf rendering which I think is an apropriate length.